### DISCUSSION TITLE
Ender-3 V1, possible?

### DISCUSSION THREAD

**Discussion by Chiller2019** [Jyers/Marlin#2064] [Q&A]
Hey,

I would flash this firmware on my Ender-3 (V1, not s/pro/v2/what else), it’s possible?

Hardware (all upgraded):
Motherboard: 4.2.2
BLTouch
Creality TouchScreen

Kind regards

**Comment by vhristenco** [comment]
It will work with the 4.2.2 MB but not with the Creality Touch screen.
